packages/core/src/html/NodeProcessor.ts

```diff
@@ -195,6 +196,9 @@ export class NodeProcessor {
       case 'question':
         this.mdAttributeRenderer.processQuestion(node);
         break;
+      case 'ul':
+        processUlNode(node);
```

Comment:
Nested lists will hit this branch again.

To prevent double processing (possibility of unwanted bugs/less headache) from processing the nested lists in one go:
- would placing this at `postProcessNode` instead work? (should be the simplest)
- or we could add a temporary attribute/flag to nested lists in the first go to skip processing them later.
- or use the original traversal process in your processing (not exactly sure how this will turn out, may be more complex)

Reply:
Fixed with `or we could add a temporary attribute/flag to nested lists in the first go to skip processing them later.`

Reply:
Let's continue this here https://github.com/MarkBind/markbind/pull/2316#discussion_r1271309228